utils/SnapPy/Snappy/AddToa.py

```diff
@@ -58,31 +62,30 @@ def add_toa_to_nc(nc: netCDF4.Dataset):
     if "time_of_arrival" in nc.variables:
         # for debugging or in special cases only
         toaVar = nc["time_of_arrival"]
-        print("overwriting existing variable time_of_arrival")
+        logging.warning("overwriting existing variable time_of_arrival")
     else:
         toaVar = nc.createVariable(
             "time_of_arrival",
             "f",
-            nc.variables[exampleVar].dimensions,
+            exampleVar.dimensions,
             zlib=True,
             fill_value=-999,
         )
     toaVar.units = "hours"
     for attr in ("grid_mapping", "coordinates"):
-        if attr in nc.variables[exampleVar].ncattrs():
-            toaVar.setncattr(attr, nc.variables[exampleVar].getncattr(attr))
+        if attr in exampleVar.ncattrs():
```

Comment:
why this change compared to the old version?

Reply:
exampleVar is now a netcdf-variable, no longer just a variable-name. See line 41-45.
Makes code shorter and better readable.